project/Dependencies.scala

```diff
@@ -44,5 +46,18 @@ object Dependencies {
       val jodaTime = "joda-time" % "joda-time" % "2.9.9"
       val slf4j = "org.slf4j" % "slf4j-api" % "1.7.18"
     }
+
+    object ScalaTest {
+      private val version = "3.2.2"
+      val test = "org.scalatest" %% "scalatest" % version % Test
+    }
+
+    object TestContainers {
+      private val version = "0.38.4"
+      val scalaTest = "com.dimafeng" %% "testcontainers-scala-scalatest" % version % IntegrationTest
```

Comment:
non blocking: purely out of interest what are theses libraries ?

Reply:
thin wrapper around `https://github.com/testcontainers/testcontainers-java/`